client/webserver/site/src/js/markets.ts

```diff
@@ -2081,17 +2151,30 @@ export default class MarketsPage extends BasePage {
    * input.
    */
   rateFieldChanged () {
-    // Truncate to rate step. If it is a market buy order, do not adjust.
```

Comment:
I think this comment is out of date since for market order `rate` gets handled by different func.